pom.xml

```diff
@@ -613,8 +613,8 @@
                         <maxmem>512m</maxmem>
                         <meminitial>256m</meminitial>
                         <encoding>UTF-8</encoding>
-
-                        <release>17</release>
+                        <!-- https://youtrack.jetbrains.com/issue/IDEA-176994 -->
+                        <!-- <release>17</release> -->
```

Comment:
This shouldn't be added to the entire build. Instead, the `pom.xml` of `jooq-checker` should contain:

```xml
<release combine.self="override"/>
```

This will remove the `--release` flag from the javac call, effectively preventing the error that this change is probably trying to address:

```
[ERROR] exporting a package from system module jdk.compiler is not allowed with --release
```